src/sentry/static/sentry/app/components/forms/inputField.tsx

```diff
@@ -27,6 +27,7 @@ class InputField<
       <input
         id={this.getId()} //TODO(Priscila): check the reason behind this. We are getting warnings if we have 2 or more fields with the same name, for instance in the DATA PRIVACY RULES
         type={this.getType()}
+        data-test-id="input-field"
```

Comment:
Instead of `test-id` attribute, can we query by `name` attribute? I think most of our fields should have a `name` and if not, they should.